src/Redist/Microsoft.ML.DnnImageFeaturizer.ModelRedist/Microsoft.ML.DnnImageFeaturizer.ModelRedist.proj

```diff
@@ -61,7 +101,14 @@
     <DownloadFilesFromUrl Items="@(ModelFiles)"/>
   </Target>
 
+  <Target Name="CopyDnnModelFiles"
+          DependsOnTargets="DownloadDnnModelFiles">
```

Comment:
This should have an `Inputs` and `Outputs` declared, so it doesn't need to run on incremental builds.